### PR TITLE
Issue #9: Nullable types should generate the proper phpdoc.

### PR DESCRIPTION
**Pull request by drupol**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: src/CodeGenerator/Assembler/NullableGetterAssembler.php [hunk 1]
```diff
@@ -59,7 +59,7 @@ public function assemble(ContextInterface $context)
                         'tags' => [
                             [
                                 'name' => 'return',
-                                'description' => $property->getType(),
+                                'description' => $property->getType() . '|null',
```
Comment (ademarco) [inline src/CodeGenerator/Assembler/NullableGetterAssembler.php]: This should result into a new generated classes too. We should commit that as part of this PR.

Comment (drupol) [reply src/CodeGenerator/Assembler/NullableGetterAssembler.php]: It's done locally, it's just that as this task was not in the sprint, I didn't dedicate time on it for now.

File: src/Type/AuxiliaryDocument.php [hunk 2]
```diff
@@ -4,15 +4,15 @@
 
 namespace OpenEuropa\EPoetry\Type;
 
-class AuxiliaryDocument extends DgtDocument
+class AuxiliaryDocument extends \OpenEuropa\EPoetry\Type\DgtDocument
```
Comment (ademarco) [inline src/Type/AuxiliaryDocument.php]: Isn't that a regression having the full namespace `\OpenEuropa\EPoetry\Type\DgtDocument`?

Comment (drupol) [reply src/Type/AuxiliaryDocument.php]: As long as this is not fixed in php-cs-fixer, we shouldn't modify them...

File: src/CodeGenerator/Assembler/NullableGetterAssembler.php (hunk 1 above)
Comment (drupol) [reply src/CodeGenerator/Assembler/NullableGetterAssembler.php]: Fixed.

**Pull request merged**